services/base.js

```diff
@@ -214,6 +219,52 @@ class BaseService {
     return result
   }
 
+  _handleError(error) {
```

Comment:
Extract into a method so `BaseStaticService` can invoke it from overridden `register()`.